src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -3737,6 +3737,9 @@ export class ZonePacketHandlers {
       case "Group.Leave":
         this.leaveGroup(server, client);
         break;
+      case "Ping":
```

Comment:
It's a real packet or crafted from us?

Reply:
No, a real packet from KOTK. Just guessing the name and structure though.